src/test/java/spimdata/OmeZarrMultiScaleV4SpimDataTest.java

```diff
@@ -0,0 +1,29 @@
+package spimdata;
+
+import lombok.extern.slf4j.Slf4j;
+import mpicbg.spim.data.SpimData;
+import org.embl.mobie.io.ome.zarr.openers.OMEZarrS3Opener;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+@Slf4j
+public class OmeZarrMultiScaleV4SpimDataTest {
```

Comment:
@tischi, this simple test checks the correct number of multi scales loaded.

Reply:
Great!